src/pages/AddPage.js

```diff
@@ -88,37 +60,6 @@ const AddPage = (props) => {
     });
   }, [lat, lng, setMapPosition]);
 
-  // LooState is the temporary loo object that hold's the user's representation of the loo
-  const [looState, setLooState] = useState(getInitialLooState());
-
-  const handleChange = (event) => {
-    setLooState({
-      ...looState,
-      [event.target.name]: event.target.value,
-    });
-  };
-
-  const handleTriStateChange = (event) => {
```

Comment:
Moved logic to https://github.com/neontribe/gbptm/pull/588/files#diff-57fe27eb6d6dcf3d0d75c740b0a0f82eR38

Reply:
Consolidated logic with https://github.com/neontribe/gbptm/pull/588/files#diff-b3d3457ad483621b96510b33c571eebfL153